tracing/README.md

```diff
@@ -149,12 +158,27 @@ is probably [`tracing-fmt`], which logs to the terminal.
 The simplest way to use a subscriber is to call the `set_global_default` function:
 
 ```rust
-#[macro_use]
 extern crate tracing;
+# pub struct FooSubscriber;
```

Comment:
We have fmt as a dev dependency right? Does that mean we could use fmt directly here? It seems like what we'd want the example to show long term

Reply:
I don't think we actually have a dev dependency on fmt, and we actually _can't_ have one until there's a crates.io release of fmt --- that would prevent us from publishing `tracing`.

Reply:
FooSubscriber it is then.